src/main/config/config_eeprom.c

```diff
@@ -92,6 +87,12 @@ void initEEPROM(void)
 
     STATIC_ASSERT(sizeof(configFooter_t) == 2, footer_size_failed);
     STATIC_ASSERT(sizeof(configRecord_t) == 6, record_size_failed);
+
+#ifdef TARGET_EEPROM_INIT
+    void targetEEPROMInit(void);
```

Comment:
declaring functions in .c file is extremely dangerous - there is no safeguard against changes in function interface, this may (and eventually will) lead to hard to find bugs ...